target/linux/ipq40xx/image/Makefile

```diff
@@ -231,6 +231,19 @@ define Device/engenius_eap1300
 endef
 TARGET_DEVICES += engenius_eap1300
 
+define Device/engenius_eap2200
+	$(call Device/FitImage)
+	$(call Device/UbiFit)
+	DEVICE_VENDOR := EnGenius
+	DEVICE_MODEL := EAP2200
+	BOARD_NAME := eap2200
```

Comment:
The BOARD_NAME variable is reserved for migrating older installations that didn't use the device-tree compatible for their board name. It shouldn't be needed for new devices. (You are installing the image from a fresh initramfs image).